openupgrade_scripts/scripts/account/18.0.1.3/post-migration.py

```diff
@@ -4,14 +4,21 @@
 from openupgradelib import openupgrade, openupgrade_180
 
 
-def replace_period_lock_date(env):
+def handle_lock_dates(env):
     openupgrade.logged_query(
         env.cr,
         """
         UPDATE res_company
         SET sale_lock_date = period_lock_date, purchase_lock_date = period_lock_date
         WHERE period_lock_date IS NOT NULL""",
     )
+    openupgrade.logged_query(
```

Comment:
In fact, I should add the condition if `account_lock` was installed or not. How we can check it?

Reply:
It's difficult. No fields, no xmlids.

Reply:
@StefanRijnhart @hbrunn any magical idea?

Reply:
nothing better than setting a flag in base's pre-migration ie in ir.config_parameter, and reading and deleting it here.

or maybe more general: do `select name, state from ir_module_module into {get_legacy_name('ir_module_module')}` at the very beginning of base's pre-migration, then we've a general way of inspecting module state before migration?